.circleci/config.yml

```diff
@@ -186,7 +186,7 @@ commands:
                   KAIROS="-kairos"
                 fi
                 TARGET_RPM=$(find $BINARY-$PLATFORM_SUFFIX/rpmbuild/RPMS/$(uname -m)/ | awk -v pat="$BINARY(d)?$KAIROS-v" '$0~pat')
-                aws s3 cp $TARGET_RPM s3://$FRONTEND_BUCKET/packages/rhel/7/kaia/
+                aws s3 cp $TARGET_RPM s3://$FRONTEND_BUCKET/packages/rhel/stream9/kaia/
```

Comment:
I think `9-stream` is more frequently used name. (unlike the docker image tag `stream9`)

- https://mirrormanager.fedoraproject.org/
- https://gist.github.com/mackmoe/6f8abafeeea71858d4c0385af8c6c0d0
- https://www.reddit.com/r/CentOS/comments/tqcuoy/all_repos_are_gone_in_centos_stream_9/?rdt=43968

Reply:
look good, I will change it